cmd/notification.go

```diff
@@ -43,7 +43,7 @@ type NotificationSys struct {
 	targetList                 *event.TargetList
 	bucketRulesMap             map[string]event.RulesMap
 	bucketRemoteTargetRulesMap map[string]map[event.TargetID]event.RulesMap
-	peerRPCClientMap           map[xnet.Host]*PeerRPCClient
+	peerClients                map[*xnet.Host]*peerRESTClient
```

Comment:
Does having map here add value? I think it can be:
`peerClients []*peerRESTClient`

Reply:
I think it is already part of `peerRESTClient`, so yes, I can change it.